steps/deploygate--upload-app-bitrise-step/1.0.0/step.yml

```diff
@@ -0,0 +1,113 @@
+title: DeployGate Upload
+summary: |
+  Distribute your in-development iOS/Android apps to your developers or testers
+description: |
+  This is a step to upload your iOS/Android application to DeployGate. DeployGate enable to share in-development apps with others instantly. For more details, please read references in [DeployGate.com](https://deploygate.com/?locale=en)
+website: https://github.com/DeployGate/upload-app-bitrise-step
+source_code_url: https://github.com/DeployGate/upload-app-bitrise-step
+support_url: https://github.com/DeployGate/upload-app-bitrise-step/issues
+published_at: 2018-08-17T17:19:54.145155459+09:00
+source:
+  git: https://github.com/DeployGate/upload-app-bitrise-step.git
+  commit: ab3ece6646dfbcbaa3bb92a50d7abbe1bcb54d05
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: curl
+  apt_get:
+  - name: git
+  - name: curl
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- api_key: ""
+  opts:
+    description: The authorization token to be used when uploading app
+    is_required: true
+    summary: User's API Key or Organization's API Key
+    title: 'DeployGate: API Key'
+- opts:
+    description: |
+      The owner name of this application
+    is_required: true
+    summary: User name or Organization name
+    title: 'DeployGate: Owner Name'
+  owner_name: ""
+- app_path: $BITRISE_APK_PATH
+  opts:
+    description: App's binary file (IPA/APK) to be uploaded
+    is_required: true
+    summary: App's binary file (IPA/APK) to be uploaded
+    title: App file path
+    value_options:
+    - $BITRISE_APK_PATH
```

Comment:
Environment variables are invalid values for the value_options items. It is because an APK or an IPA path might comes from another env, or the envs are just might not set. 

Please remove value options in this case and leave the default value empty, so the user will need to fill itt accordngly.

Thank you for the PR!